test/utils_spec.js

```diff
@@ -1107,6 +1107,21 @@ describe("utils", function () {
             },
             `text:Arial_18_bold_italic_letter_spacing_4_line_spacing_2:${text_encoded}`,
           ],
+          [
+            "text with text font_antialiasing and font_hinting",
```

Comment:
@strausr , as discussed, let's add a separate test for the antialiasing and for hinting